cloudcompose/cluster/aws/ebs.py

```diff
@@ -54,6 +54,10 @@ def _create_volume_config(self, volume, default_device, use_snapshots):
                 "VolumeType": volume.get("volume_type", "gp2")
             }
         }
+
+        if volume_config['Ebs']['VolumeType'] == 'io1':
+            volume_config['Ebs']['Iops'] = volume.get("iops", volume_config['Ebs']['VolumeSize']*30)
```

Comment:
Do we always want to default to the maximum?  That seems like an expensive default.  It might be better to default to the minimum, right?